### PR TITLE
[DX-1728]-Fixed 5.3 RNs

### PR DESCRIPTION
**Pull request by sharadregoti**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

[Jira](https://tyktech.atlassian.net/browse/DX-1728)

---

### Preview Link

<br>

[Preview](https://deploy-preview-5666--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.3/)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
documentation

___

### **Description**
- Updated the upgrade instructions in the release notes for Tyk Dashboard and Tyk Gateway versions 5.3.x.
- Simplified the upgrade process by removing references to older versions and providing direct upgrade instructions for each minor release.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Update upgrade instructions for Tyk Dashboard 5.3.x releases</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md

<li>Updated upgrade instructions for versions 5.3.1 to 5.3.6.<br> <li> Replaced specific version references with a more general instruction <br>format.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5666/files#diff-14b1be0b2206c9ee21cb1002b55361919932e84558f614c4342f05b8eb5431f8">+8/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Revise upgrade instructions for Tyk Gateway 5.3.x releases</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md

<li>Updated upgrade instructions for versions 5.3.1 to 5.3.7.<br> <li> Simplified upgrade path by removing older version references.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5666/files#diff-ead598197a92e18014f4f7c35c341621948fb6cdf66d45c1dd05de21bc2b35aa">+7/-21</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Review by andyo-tyk — APPROVED** [review]
LGTM, thanks.

**Review by Eopayemi — APPROVED** [review]

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ If moving from a version of Tyk older than 5.3.0 please read the explanation pro
 There are no deprecations in this release.
 
 ### Upgrade Instructions
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first [upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
+If you are upgrading to 5.3.6, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.6, please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 2]
```diff
@@ -269,7 +269,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first [upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
+If you are upgrading to 5.3.5, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.5, please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 3]
```diff
@@ -340,7 +340,8 @@ There are no breaking changes in this release, however, if moving from a version
 There are no deprecations in this release.
 
 ### Upgrade Instructions
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first [upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
+If you are upgrading to 5.3.4, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.4 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 4]
```diff
@@ -453,7 +454,7 @@ There are no breaking changes in this release, however, if moving from a version
 There are no deprecations in this release.
 
 ### Upgrade Instructions
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first [upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
+If you are upgrading to 5.3.3, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.3 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 5]
```diff
@@ -583,7 +584,7 @@ There are no deprecations in this release.
 
 
 ### Upgrade Instructions
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first [upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
+If you are upgrading to 5.3.2, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.2 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 6]
```diff
@@ -727,7 +728,8 @@ There are no breaking changes in this release, however if moving from a version
 There are no deprecations in this release.
 
 ### Upgrade Instructions
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first [upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
+If you are upgrading to 5.3.1, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.1, please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 7]
```diff
@@ -46,9 +46,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.7, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.7 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 8]
```diff
@@ -170,9 +168,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.6, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.6 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 9]
```diff
@@ -364,9 +360,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.5, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.5 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 10]
```diff
@@ -893,9 +881,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.1, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.1 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 11]
```diff
@@ -749,9 +739,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.2, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.2 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 12]
```diff
@@ -582,9 +574,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.3, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.3 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 13]
```diff
@@ -507,9 +501,7 @@ There are no deprecations in this release.
 
 ### Upgrade Instructions
 
-If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should first
-[upgrade to 5.3.0](#upgrade-5.3.0) and then upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk)
-section for detailed upgrade instructions.
+If you are upgrading to 5.3.4, please follow the detailed [upgrade instructions](#upgrading-tyk).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]:
```suggestion
When upgrading to 5.3.4 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

**Review by letzya — APPROVED** [review]

**Pull request merged**